scenarios/academic/KDD2020-tutorial/reco_cpu_kdd.yaml

```diff
@@ -0,0 +1,37 @@
+# 
```

Comment:
Same comment - do we need this to be merged?

Reply:
I think so